### DISCUSSION TITLE
x.stats undefined

### DISCUSSION THREAD

**Discussion by Dedsd** [vasqued2/ha-teamtracker-card#77] [Q&A]
I use your table NHL example, for soccer table. It was working fine(months), until the last week wich I've seen this message:
![image](https://github.com/vasqued2/ha-teamtracker-card/assets/71463162/a07c7c0b-5d0d-4a69-9296-851024c1fed8)

**Comment by vasqued2** [comment]
This isn't a Team Tracker issue.  If you post on this thread or to the linked repository, you should find someone to help you.

[https://community.home-assistant.io/t/sports-standings-and-scores/547094](https://community.home-assistant.io/t/sports-standings-and-scores/547094)
